neqo-http3/src/lib.rs

```diff
@@ -543,6 +544,47 @@ trait HttpRecvStreamEvents: RecvStreamEvents {
     fn extended_connect_new_session(&self, _stream_id: StreamId, _headers: Vec<Header>) {}
 }
 
+// See https://www.w3.org/TR/webtransport/#send-stream-stats.
+#[derive(Debug, Clone)]
+pub struct SendStreamStats {
+    pub timestamp: Duration,
```

Comment:
Is this really a `Duration`?  What is the reference time?

Reply:
I'll remove this.